docs/CONFIG-VARS.md

```diff
@@ -293,3 +294,9 @@ postgres_servers = {
   }
 }
 ```
+
+## Monitoring
```

Comment:
We call this monitoring; however, this was already part of the code base in `create_gke_monitoring_service` what/how will we distinguish these items and keep the confusion down with our customers? I know AWS has a prometheus offering and Azure has their version in preview, Is prometheus a variant of monitoring? and/or a solution being offered as its the industry go to?

Reply:
Summary of : https://cloud.google.com/stackdriver/docs/solutions/gke

The main difference is that `gke_monitoring_service` (Cloud Monitoring service add-on) is used to to monitor both the the system and workload components of the cluster. However I just found out that as of 1.24 workload monitoring is deprecated and Google recommends that the Google Cloud Managed Service for Prometheus is used monitor and alert on your workloads. Quote on bottom.

Even through the monitoring is split up between the GKE add on and Prometheus all the data will still flow into the GCP monitoring service here https://console.cloud.google.com/monitoring

Another behavior that I saw is that with `enable_managed_prometheus` and `gke_monitoring_service` both enabled the user needs to explicitly specify which components the `gke_monitoring_service` (the add-on) will monitor, this requires one additional variable `gke_monitoring_enabled_components` that I am in process of adding.

I saw this in the GKE details page

>Cloud Monitoring is a Google Kubernetes Engine (GKE) addon that collects metrics emitted by your applications and by GKE infrastructure. Learn more 

>System monitoring includes over 40 platform metrics such as CPU, memory, and storage for essential system components. Cloud Monitoring does not charge for the ingestion of system metrics. Learn more 

>Workload monitoring is deprecated. Workload monitoring is removed in GKE 1.24 and later. Workload monitoring is replaced by Google Cloud Managed Service for Prometheus, which is Google's recommended way to monitor Kubernetes applications by using Cloud Monitoring. Learn more